scripts/scil_tractogram_dps_math.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Add, extract or delete information to each streamline from a tractogram
+file. Can be for example SIFT2 weights, processing information, bundle IDs,
+tracking seeds, etc.
+
+Input and output tractograms must always be .trk to simplify filetype checks
+for each operation.
+"""
```

Comment:
Before the end of the help, can you provide an example command line for this script. I'm new to this and I don't know how I would call it. Not sure what a dps_key is ... One or two examples would help. Thanks